### PR TITLE
Add Scopescan.ai to the explorer doc section (follow PR 226)

### PR DESCRIPTION
**Pull request by RedHorse823**
**What changed? Why?**

I add Scopescan, a data analytic platform to the block explorer documentation page.

This is because Scopescan also support to check Base transactions & tokens & addresses, they also have other nice features like:

- moneyflow tool to visualize token flows
- Token holder graph to visualize token holder distributions
- Smartmoney dashboard to check who made the most money on Base

**Notes to reviewers**

_This is a new PR followed by the previously closed PR 226_

Scopescan is power by 0xscope, they received legit fundings from OKX and Hashkey.

I believe Scopescan can help a lot with Base users with their data and tooling ability. That's why i propose this change. Hope it get approved, thanks!

scan.0xscope.com is their platform link, you can try and decide.

### CODE REVIEW

**Review by zencephalon — APPROVED** [review]

**Comment by cb-heimdall** [comment]
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>1/2</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>1</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>2</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>

**Review by wbnns — APPROVED** [review]
ACK

File: apps/base-docs/docs/tools/block-explorers.md [hunk 1]
```diff
@@ -76,6 +76,11 @@ A testnet explorer for [Base Goerli](https://base-goerli.dex.guru/) is also avai
 
 [Routescan](https://superscan.network/) superchain explorer allows you to search for transactions, addresses, tokens, prices and other activities taking place across all Superchain blockchains, including Base.
 
+
+## Scopescan
+
+[Scopescan](https://scan.0xscope.com/home?network=base) supports Base on their platform. Base voyagers can check Base transaction records, addresses, tokens, and other traits on their platform. Scopescan also supports visualizing capital flows and checking token holder graphs on Base.
```
Comment (wbnns) [inline apps/base-docs/docs/tools/block-explorers.md:82]: @RedHorse823 

Thanks for this PR! Can you please update the language to:
```
[Scopescan](https://scan.0xscope.com/home?network=base) enables people to check Base transaction records, addresses, tokens, and other traits on their platform. Scopescan also supports visualizing capital flows and checking token holder graphs on Base.
```

**Review by wbnns — CHANGES_REQUESTED** [review]
@RedHorse823 

Requested a small change on the copy. 🙏

**Comment by cb-heimdall** [comment]
Review Error for wbnns @ 2024-10-17 22:43:30 UTC
User failed mfa authentication, see [go/mfa-help](http://go/mfa-help)

**Review by wbnns — CHANGES_REQUESTED** [review]
[Requested change](https://github.com/base-org/web/pull/547/files#r1805532650)

**Comment by wbnns** [comment]
@RedHorse823 Please let us know if you're still interested in pursuing these changes and we can reopen. 👍

**Pull request closed without merging**